ios/PurchasesHybridCommon/PurchasesHybridCommon/EntitlementInfo+HybridAdditions.swift

```diff
@@ -1,49 +1,42 @@
 //
-//  EntitlementInfoExtensions.swift
+//  EntitlementInfo+HybridAdditions.swift
 //  PurchasesHybridCommon
 //
 //  Created by Andrés Boedo on 4/13/22.
 //  Copyright © 2022 RevenueCat. All rights reserved.
 //
 
 import Foundation
-import Purchases
+import RevenueCat
 
-@objc public extension Purchases.EntitlementInfo {
-
-    @objc var dictionary: [String: Any] {
-
-        let latestPurchaseNSDate = latestPurchaseDate as NSDate
```

Comment:
we're now using `Date` instead of `NSDate` since this is coming from a Swift framework, so the extra conversion isn't needed anymore